src/ringct/rctSigs.cpp

```diff
@@ -32,8 +32,9 @@
 #include "common/perf_timer.h"
 #include "common/threadpool.h"
 #include "common/util.h"
-#include "rctSigs.h"
+#include "bulletproofs.h"
 #include "cryptonote_basic/cryptonote_format_utils.h"
+#include "rctSigs.h"
```

Comment:
Did you bring this include to the last because the ordering matters?

Reply:
It was just missing AFAICT.

Reply:
I'm not sure what you mean and if I made my intent clear. My intent was: isn't it sufficient to just add include for bulletproofs.h, without moving the line of include for rctSigs.h (which adds an unnecessary change).

Reply:
I see now. I did not notice it had moved, I'll put it back where it was. I thought it was missing all along.